src/pages/WordFrequency/WordFrequency.tsx

```diff
@@ -31,7 +31,8 @@ export interface WordFrequencyParams {
 }
 
 export const WordFrequency = (props: RouteComponentProps<WordFrequencyParams>) => {
-  const [script] = H.useLocalStorageState<string>(PSC.Script.RO, 'currentScript')
+  const [scriptRaw] = H.useLocalStorageState<string>(PSC.Script.RO, 'currentScript')
+  const script = scriptRaw === 'xx' ? 'Latn' : scriptRaw

```

Comment:
should have been dealt with much before as part of substep 1  in step 1, my bad. (the drawer in the word frequency section behaved strange for locale xx.)